apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -170,7 +170,8 @@ const InfiniteMobileTeamsTab: FC<InfiniteMobileTeamsTabProps> = (props) => {
     },
     {
       refetchOnWindowFocus: true,
-      staleTime: 1 * 60 * 60 * 1000,
+      refetchOnMount: true,
+      staleTime: 0,
```

Comment:
Need this to render new event type when user clicks back from event type and doesn't reload.